mass_editing/migrations/14.0.1.0.0/post-migrate.py

```diff
@@ -0,0 +1,38 @@
+# Copyright (C) 2020 - Iván Todorovich <ivan.todorovich@gmail.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from openupgradelib import openupgrade
+
+
+@openupgrade.migrate()
+def migrate(env, version):
+    if openupgrade.table_exists(env.cr, "mass_editing"):
+        migrate_mass_editing(env)
+
+
+def migrate_mass_editing(env):
```

Comment:
You need to put this method before the method that invokes it.

Reply:
done!